cli/medperf/comms/rest.py

```diff
@@ -114,6 +114,61 @@ def __req(self, url, req_func, **kwargs):
                 "remember to provide the server certificate through --certificate"
             )
 
+    def __get_list(
+        self,
+        url,
+        num_elements=None,
+        page_size=config.default_page_size,
+        offset=0,
+        binary_reduction=True,
```

Comment:
Do we need to keep it as True? From one of our earlier discussions, return-code != 200 can happen because of reasons other than the page limit issue

Reply:
Changed the default to False. After the discussion I understand the concerns with defaulting to True, and I can see this should only be used on extraordinary cases where we know dynamically changing the size of the page will help overcome an issue.